elements/lisk-api-client/test/api_client.spec.ts

```diff
@@ -209,56 +194,45 @@ describe('APIClient module', () => {
 						engine: '+https://github.com/LiskHQ/lisk-hub',
 					},
 				});
-				return expect(apiClient)
-					.to.have.property('headers')
-					.and.eql(customHeaders);
+				return expect(apiClient.headers).toEqual(customHeaders);
 			});
 
 			it('should not set User-Agent header when client options were not given', () => {
 				apiClient = new APIClient(defaultNodes, {
 					nethash: testnetHash,
 				});
-				return expect(apiClient.headers).to.not.have.property('User-Agent');
+				return expect(apiClient.headers).not.toHaveProperty('User-Agent');
 			});
 		});
 
 		describe('nodes', () => {
 			it('should have nodes supplied to constructor', () => {
-				return expect(apiClient)
-					.to.have.property('nodes')
-					.and.equal(defaultNodes);
+				return expect(apiClient.nodes).toBe(defaultNodes);
```

Comment:
expect(apiClient).toHaveProperty('nodes', defaultNodes);

Reply:
I think they are essentially the same?

Reply:
Sure, second is more readable as it refer that it is an object.

Reply:
Will try to use it in other PRs